test/integration/base_integration_test.h

```diff
@@ -369,11 +369,14 @@ class BaseIntegrationTest : protected Logger::Loggable<Logger::Id::testing> {
 protected:
   void setUdpFakeUpstream(bool value) { upstream_config_.udp_fake_upstream_ = value; }
   bool initialized() const { return initialized_; }
-  const FakeUpstreamConfig& upstreamConfig() {
-    // TODO(alyssawilk) make enable_half_close_ private and remove this.
-    upstream_config_.enable_half_close_ = enable_half_close_;
-    return upstream_config_;
-  }
+
+  // Right now half-close is set globally, not separately for upstream and
+  // downstream.
+  void enableHalfClose(bool value) { upstream_config_.enable_half_close_ = value; }
+
+  bool enableHalfClose() { return upstream_config_.enable_half_close_; }
```

Comment:
Not sure if we want to name these two differently (to have different names for getter and setter). But I'm OK with this.